docker-compose.yml

```diff
@@ -36,6 +36,14 @@ services:
       - 'ELASTIC_USERNAME=latitude'
       - 'ELASTIC_PASSWORD=secret'
 
+  database:
```

Comment:
Maybe is a good time to rename now this as talked offline

Reply:
hmm good point